src/game/gameOverModal.scss

```diff
@@ -0,0 +1,47 @@
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.8);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: white;
+  &__content {
+    max-width: 90vw;
+    max-height: 90vh;
+
+    h2 {
+      border-bottom: 5px solid white;
+      line-height: 1.4;
+      font-size: 48px;
+      margin: 0;
+      @media screen and(min-width: 767px) {
+        font-size: 144px;
+      }
+    }
+    h3 {
+      font-size: 24px;
+      @media screen and(min-width: 767px) {
+        font-size: 48px;
+      }
+    }
+  }
+  &__actions {
+    margin-top: 4rem;
+    a {
+      display: block;
+      font-size: 32px;
+      color: white;
```

Comment:
Move it to a variable. And for consistency, I think it will be better if you will use here `HEX` without shorthand.

Reply:
I don't think that we need use variable for basic color like this. IMO HEX is enough